### PR TITLE
PE-313 Add back to top link

### PR DESCRIPTION
**Pull request by radmosley**

### CODE REVIEW

File: web/themes/custom/westy/src/scss/components/_back-to-top.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#back-to-top {
+    position: fixed;
+    bottom: 4rem;
+    right: 9rem;
+    z-index: $zindex-popover;
+  }
```
Comment (joshuami) [inline web/themes/custom/westy/src/scss/components/_back-to-top.scss:6]: Needs a newline.

File: web/themes/custom/westy/assets/css/westy.style.css [hunk 2]
```diff
@@ -1 +1,12300 @@
...
+@charset "UTF-8";
```
Comment (joshuami) [inline web/themes/custom/westy/assets/css/westy.style.css:1]: You need to `npm run production` to get the minified versions of CSS and JS.

**Review by joshuami — CHANGES_REQUESTED** [review]
Couple of items to fix.

File: web/themes/custom/westy/src/scss/components/_back-to-top.scss (hunk 1 above)
Comment (radmosley) [reply web/themes/custom/westy/src/scss/components/_back-to-top.scss]: Add newline to file

File: web/themes/custom/westy/assets/css/westy.style.css (hunk 2 above)
Comment (radmosley) [reply web/themes/custom/westy/assets/css/westy.style.css]: ran the command to minify css and js

File: web/themes/custom/westy/src/scss/components/_back-to-top.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (joshuami) [inline web/themes/custom/westy/src/scss/components/_back-to-top.scss:7]: Needs a newline per Github.

**Review by joshuami — CHANGES_REQUESTED** [review]
@radmosley , Github is still showing one file needs a newline.

File: web/themes/custom/westy/src/scss/components/_back-to-top.scss (hunk 1 above)
Comment (joshuami) [reply web/themes/custom/westy/src/scss/components/_back-to-top.scss]: It's odd that Github is still seeing that as not having a newline. 🤔

File: web/themes/custom/westy/src/scss/components/_back-to-top.scss (hunk 3 above)
Comment (radmosley) [reply web/themes/custom/westy/src/scss/components/_back-to-top.scss:8]: file updated with new line

Comment (joshuami) [reply web/themes/custom/westy/src/scss/components/_back-to-top.scss:8]: This is odd. It is still showing the need for a newline.

2022-08-26 at 4 59 PM

@kxwang , @jookyg , or @RichardDavies have you seen this before?

**Comment by radmosley** [comment]
It still shows the file as needing a new line in "files changed" but if I open the file in the github ui the new line is there.

**Review by RichardDavies — APPROVED** [review]

File: web/themes/custom/westy/src/scss/components/_back-to-top.scss (hunk 1 above)
Comment (RichardDavies) [reply web/themes/custom/westy/src/scss/components/_back-to-top.scss]: @radmosley 's latest changes removed the newline warning for me.

**Pull request merged**